src/main/java/ch/njol/skript/variables/SerializedVariable.java

```diff
@@ -11,15 +11,15 @@ public class SerializedVariable {
 	/**
 	 * The name of the variable.
 	 */
-	public final String name;
+	private final String name;
 
 	/**
 	 * The serialized value of the variable.
 	 * <p>
 	 * A value of {@code null} indicates the variable will be deleted.
 	 */
 	@Nullable
-	public final Value value;
+	private final Value value;
```

Comment:
Do you think it's worth deprecating/marking these as internal instead of immediately going private or is this too niche to really matter